test/hygiene.jl

```diff
@@ -5,7 +5,6 @@
 
 # hygiene.jl
 # Make sure that our macros have good hygiene
-using Base.Test
 
 module M
```

Comment:
I though the issue occurs when the expressions are at the global scope. Is this testing what we intend when the code is inside a model?

Reply:
According to [Julia docs](https://docs.julialang.org/en/v1/manual/variables-and-scoping/), global scopes are on modules and REPL

Reply:
Ok, LGTM. Could you add a TODO to add actual test statements below? :)

Reply:
What should the TODO say ? That it needs to be in global scope and we use a module for that ?